offline license utilization reporting (#25844)
hashicorp/nomad: website/content/docs/commands/operator/utilization.mdx

---
layout: docs
page_title: nomad operator utilization command reference
description: |-
  The "operator utilization" generates utilization reporting bundles for Nomad Enterprise users.
---

# `nomad operator utilization` command reference

The `operator utilization` command allows Nomad Enterprise users to generate
utilization reporting bundles. If you have disabled automated reporting, use

> this command to manually generate the report and send it to HashiCorp. If no
> snapshots were persisted in the last 24 hrs, Nomad takes a new snapshot.

Reviewer: ```suggestion
this command to manually generate the report and send it to HashiCorp. Nomad
takes a new snapshot when there are no snapshots from the past twenty-four
hours.
```

If ACLs are enabled, this command requires a token with the `operator:write`
capability.

<Note title="Nomad Enterprise">

Refer to the [manual license utilization
reporting](/nomad/docs/enterprise/license/manual-reporting) page to learn more
about reporting your Nomad Enterprise license utilization.

</Note>

## Command Options

- `-message` `(string; "")` - Provide context about the conditions under which
  the report was generated and submitted. This message is not included in the

>   utilization bundle but will be included in the Nomad server logs.

Reviewer: ```suggestion
  utilization bundle but is included in the Nomad server logs.
```
